scripts/manilananny.py

```diff
@@ -119,7 +119,14 @@ def list_shares(self, status=None):
             search_opts = {'all_tenants': 1}
             if status:
                 search_opts.update({'status': status})
-            return self.manilaclient.shares.list(search_opts=search_opts)
+
+            shares = []
+            offset = 0
+            for i in range(10):
```

Comment:
Please use the `with_count` parameter to determine the total number of shares - we have already more than 10k shares in our big regions.

Reply:
done